io.frama.tractor.carburetor.yaml

```
@@ -0,0 +1,59 @@
+app-id: io.frama.tractor.carburetor
+runtime: org.gnome.Platform
+runtime-version: '45'
+sdk: org.gnome.Sdk
+command: carburetor
+finish-args:
+  # X11 + XShm access
+  - --share=ipc
+  - --socket=fallback-x11
+  # Wayland access
+  - --socket=wayland
+  # DRI access for OpenGL rendering
+  - --device=dri
+  # Needs to talk to the network:
+  - --share=network
+  # Access to GNOME Virtual FileSystem
+  - --filesystem=xdg-run/gvfsd
+modules:
+  - python3-requirements.yaml
+  - name: libevent
+    sources:
+      - type: archive
+        url: https://github.com/libevent/libevent/releases/download/release-2.1.12-stable/libevent-2.1.12-stable.tar.gz
+        sha256: 92e6de1be9ec176428fd2367677e61ceffc2ee1cb119035037a27d346b0403bb
+  - name: tor
+    config-opts:
+      - --disable-asciidoc
+      - --disable-manpage
+      - --disable-html-manual
+      - --disable-unittests
+    sources:
+      - type: archive
+        url: https://github.com/torproject/tor/archive/refs/tags/tor-0.4.7.13.tar.gz
+        sha256: a17b0e085a9927dffd414ed54b0c0f16caf1e8efdaba72878575c005c67856a1
+  - name: tractor
+    buildsystem: simple
+    build-commands:
+      - pip3 install --verbose --exists-action=i --no-index --find-links="file://${PWD}"
+        --prefix=${FLATPAK_DEST} . --no-build-isolation
+      - install -Dm644 tractor/tractor.gschema.xml -t ${FLATPAK_DEST}/share/glib-2.0/schemas
+      - glib-compile-schemas ${FLATPAK_DEST}/share/glib-2.0/schemas
+      - cp ${PWD}/tractor/SampleBridges ${FLATPAK_DEST}/lib/python3.11/site-packages/tractor/
```

```suggestion
      - cp ${PWD}/tractor/SampleBridges ${FLATPAK_DEST}/lib/python$(python -c 'import sys; print("%s.%s" %sys.version_info[0:2])')/site-packages/tractor/
```

This won't break in between runtime bumps.